ui/cypress/e2e/sampleControls.cy.js

```diff
@@ -0,0 +1,41 @@
+/* global cy, it, describe, beforeEach, Cypress */
+
+describe('3-click centring', () => {
+  beforeEach(() => {
+    cy.login();
+    cy.findByRole('link', { name: 'MXCuBE-Web (OSC)' }).should('be.visible');
+    cy.takeControl();
+  });
+
+  it('3-click centring should not work without sample', () => {
+    cy.clearSamples();
+    cy.contains('button', '3-click centring').click();
+    cy.findByRole('alert', 'Error: There is no sample mounted').should(
+      'be.visible',
+    );
+  });
+
+  it('Each click is rotating the sample by 90 degrees', () => {
+    cy.mountSample();
+    cy.contains('button', '3-click centring').click();
+    /* eslint-disable-next-line promise/catch-or-return */
+    cy.get('.form-control[name="diffractometer.phi"]')
+      .invoke('val')
+      /* eslint-disable-next-line promise/prefer-await-to-then */
```

Comment:
You could disable those two warnings for all Cypress specs in https://github.com/mxcube/mxcubeweb/blob/develop/ui/.eslintrc.js#L59

Reply:
Thanks for the comment, you are right that would be a better solution!